src/server.ts

```diff
@@ -1,22 +1,14 @@
-import fastify from 'fastify';
-import { config } from './config/env';
-import { prettyLog } from './utils/prettyLog';
+import { buildServer } from "@/server/";
 
-const server = fastify({
-  logger: {
-    transport: prettyLog,
```

Comment:
One doubt... This piece is creating a log struct with a configuration that prettify logs?